flutter-studio/src/io/flutter/profiler/FlutterStudioProfilersView.java

```diff
@@ -295,6 +321,42 @@ private void initializeStageUi() {
     resetSnapshotStatistics.setToolTipText("Reset Snapshot Statistics");
     leftToolbar.add(resetSnapshotStatistics);
 
+    // Memory Profiler setting button.
+    JBPopupMenu menuBar = new JBPopupMenu();
+
+    // TODO(terry): When LafIconLookup in all IDE builds use LafIconLookup.getIcon("checkmark")
+    Icon enabledCheckmark = ProfilerCheckMark;
+    Icon checkMarkSpacer = new EmptyIcon(enabledCheckmark.getIconWidth(), enabledCheckmark.getIconHeight());
+
+    JBMenuItem rssEnabledMenu = new JBMenuItem(VIEW_RSS_STATISTICS_MENU_ITEM, checkMarkSpacer);
+    rssEnabledMenu.addActionListener(event -> {
+      FlutterStudioMonitorStageView view = (FlutterStudioMonitorStageView)(this.getStageView());
+      displayRSSInformation = !displayRSSInformation;   // Toggle the state.
```

Comment:
Can you reuse the existing checkbox menu item class uses in the inspector menu instead?

Reply:
Yes